content/v0.80.0.md

```diff
@@ -0,0 +1,466 @@
+        |- scaler/rgb3x
+        |- scaler/scan2x
+        |- scaler/scan3x
+        |- scaler/tv2x
+        `- scaler/tv3x
+```
+
+<details>
+  <summary>Log of 0.80's pull requests involving additions and improvements:</summary>
+- #2137: Add a subset of OpenGL shaders from Tyrells repo
+- #1996: Add direct support for Windows 3.1 seamless mouse driver by javispedro
+- #2075: Add libraries to support TrueType font rendering
+- #2099: Add more cleanup on audio device tear-down
+- #2062: Add mouse related hints to title bar
+- #2036: Add release drafter support files
+- #2078: Add screenshots for rendered output (#2073)
+- #2122: Add `--working-dir` command line option
+- #2067: Allow to cancel interactive mouse mapping with a keyboard
+- #2025: Bump actions/cache from 2 to 3.0.11
+- #2028: Bump actions/download-artifact from 2 to 3
+- #2026: Bump actions/upload-artifact from 2 to 3
+- #2027: Bump fountainhead/action-wait-for-check from 1.0.0 to 1.1.0
+- #2123: Bump microsoft/setup-msbuild from 1.1 to 1.2
+- #2129: Bump microsoft/setup-msbuild from 1.2 to 1.3
+- #2108: Check file-descriptor dependency for Enet, slirp, and ManyMouse
+- #2096: Cleanup a handful of build issues for clang 10.x and 11.x
+- #2084: Cleanup issues in ReelMagic flagged by Coverity
+- #2104: Cleanup overcomplicated data conversions in Unicode support
+- #2064: Cleanup string IDs of DOS programs
+- #2076: Compiler warning fix in ManyMouse library
+- #2037: Contribute a script to convert from YouTube chapters to CD-DA
+- #2127: Customise the looks of the macOS DMG distribution disk image
+- #2101: Debloat the binary: use more heap storage and retire the software scalar
+- #2124: Drop architecture spec from setup-msbuild GitHub action
+- #2105: Enable DPI scaling on Windows
+- #2040: Fix a couple CWE issues in ManyMouse
+- #2068: Fix sentence starting with lower case in MOUNT command help
+- #2103: Generate linker map file by default
+- #2020: Implement MORE.COM command
+- #2077: Import Jon Dennis's ReelMagic emulation package (#1112)
+- #1992: Improve detection of atomics, testing of speexDSP, and cleanup README
+- #2065: Improve interactive mouse mapping
+- #2095: Improve management of audio devices
+- #2085: Look for libXInput dependency
+- #2060: Minor mouse enhancements
+- #2000: Mouse mapper and configuration mechanism
+- #2086: Refactor ReelMagic player with per-stream handle registration
+- #2090: Refactor the ReelMagic audio FIFO class
+- #2130: simplifies file search in virtual drive
+- #2045: Split contributed CD-DA scripts into fetch vs. cue functions
+- #2092: Sync reSIDfp with upstream
+- #2031: Update CI against latest deprecations and OS versions
+- #2024: Update the build system wraps and automate CI action maintenance
+- #2069: Update Tracy to v0.9
+- #2097: Update wraps and PVS Studio
+- #2116: Use a `sdd::unique_ptr` to manage the video memory
+- #2079: Use MORE engine for help output
+- #2117: Use shared pointers to manage the virtual files
```

Comment:
How about A-Z sorting all the logs? Now it's random order, I'd say A-Z sorting it would at least order them by the PR number.

Reply:
got 'em.